grails-app/controllers/au/org/emii/portal/LayerController.groovy

```diff
@@ -25,6 +25,12 @@ class LayerController {
 
     static allowedMethods = [save: "POST", update: "POST", delete: "POST"]
 
+    static final HTTP_200_OK = 200
```

Comment:
Perhaps extract this to a more common file? I bet we can use it in many more places...